app/middlewares/verifyCookieToken.js

```diff
@@ -0,0 +1,27 @@
+const tokenUtil = require('../auth/token');
+
+/**
+ * verifies cookie token for each parking spot, and returns info back from cookie to req.spotInfo.
+ * @param req
+ * @param res
+ * @param next
+ * @returns {Promise<void>}
+ */
+async function verifyCookieToken(req, res, next) {
+    try {
+        const token = req.cookies.token;
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443778)
```suggestion
    const token = req.cookies.token;
```